e2e/utils.go

```diff
@@ -319,6 +319,12 @@ func createRBDStorageClass(c kubernetes.Interface, f *framework.Framework, param
 		sc.Parameters[k] = v
 	}
 	sc.Namespace = cephCSINamespace
+	for k, v := range scOptions {
+		if k == "volumeBindingMode" && v == "WaitForFirstConsumer" {
```

Comment:
Are these strings not available as constants?

Reply:
I cannot find a reusable constant for `volumeBindingMode`.

For `WaitForFirstConsumer` there is a typed string constant (which is used just below this as `scv1.VolumeBindingWaitForFirstConsumer`), but this makes them non type compliant causing linter failures.

Also, this means we need to map for `scOptions` that this function takes, be a map with different keys and values?

If you have an alternative I can code that up, else I am out of options on this.

Reply:
declared `type VolumeBindingMode string` constants can be used here @ShyamsundarR instead of hard-coding these.

Reply:
As stated, I am using these constants just below, but due to type difference not able to cleanly use them here. I would assume this does not stop the PR from being merged, though. If it does, please let me know.